scripts/build-native.sh

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
+
+#
+# Simple Wrapper to build a native binary for your sbt application.
+#
+# Configuration:
+# * Target:
+#   * `native` - Builds the binary for the machine arch (Requires GraalVM and Oracle JDK 8)
+#   * `docker` - Builds the binary for linux using a docker (Requires Docker)
+#
+# Setup GraalVM and Oracle JDK 8:
+#   * Install GraalVM (https://www.graalvm.org/docs/getting-started/).
+#   * Install Oracle JDK 8 (http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html).
+#   * Add GraalVM native-image bin to the beginning of your PATH.
+#   * Set Oracle JDK 8 as your JAVA_HOME, JDK_HOME and JRE_HOME.
+#   * Add Oracle JDK 8 bin to the beginning of your PATH.
+#
+# Example:
+# ./scripts/publish-native.sh -n codacy-plugins-test -m codacy.plugins.DockerTest -t docker -s 2.12 1.0.0
+#
+
+set -e
+
+SCALA_VERSION="2.11"
```

Comment:
You can remove SCALA_VERSION since it is not needed anymore.

Reply:
I removed it.